rts/lra/lra-coordinator-jar/src/main/java/io/narayana/lra/coordinator/api/Coordinator.java

```diff
@@ -112,6 +113,7 @@
     @APIResponse(description = "The LRA",
         content = @Content(schema = @Schema(type = SchemaType.ARRAY, implementation = LRAData.class))
     )
+    @RolesAllowed({"admin","participant"})
```

Comment:
We have to hardcode it in all the endpoints (where ever we need security), because we have to make this code common (independent of server like wildfly, quarkus or any other product that comes in future), so that it could be used by every one.

@RolesAllowed only allows to pass constant as its argument.

Reply:
For now there is no option for reconfiguration of this roles. We will have to come up with the possible `roles` that user would follow in his application. It could just be related to LRA.

If you could suggest what all roles could we use in all the endpoints.
Here the problem I see is that, there are many endpoints, and foreach endpoint we need to define the set of allowed roles. I am not clear, like who will decide this.

Reply:
I agree It's hard to define right now. 
I liked the @mmusgrov point on the meeitng.

Let's have a some basic granularity here. Let's create one role which may everything (`admin` propably) and then one role which has only read-only permissions (like it can't start, cancel etc. but it can get e.g. status) and a role which may enlist and renew timelimit but it can't start and end the lra. What do you think? Would you have some other idea on how to make the roles more granular?

Reply:
This make sense. But from user's perspective, having too many `roles` would be difficult to manage and that too only for LRA, there would be other roles already present in user's application, maybe if we could have only two roles for now, as @mmusgrov mentioned in the meeting, would be easy/simple approach.

You are suggesting that `participant` role to have only `view` rights. Here maybe we could provide some more rights to `participant` role?

Reply:
I don't think more possible roles makes some issue. It rather provides more possibilities for configuration. The user just decides which role to use and he does not need to manage anything else - do you want to read/write -> take admin, are you ok with view -> take viewer, do you want to enlist -> take participant.
But if you think differently it's fine for me for now.